uaclient/cli.py

```diff
@@ -1619,6 +1620,64 @@ def action_refresh(args, *, cfg: config.UAConfig):
     return 0
 
 
+def is_contract_changed(cfg: config.UAConfig) -> bool:
+    orig_token = cfg.machine_token
+    orig_entitlements = cfg.entitlements
+    machine_token = orig_token["machineToken"]
+    contract_id = orig_token["machineTokenInfo"]["contractInfo"]["id"]
```

Comment:
Let's use `get` calls here to access the dict keys, just so we can be safe that status will not break if any of those keys is missing for some reason (even though this should be a corner case situation)

Reply:
Yeah, nice catch.